pkg/spec/operation.go

```diff
@@ -341,80 +332,149 @@ func CloneOperation(op *spec.Operation) (*spec.Operation, error) {
 	return &out, nil
 }
 
-func generateAuthBearerScheme(operation *spec.Operation, bearerToken string, sdName string) (*spec.Operation, *spec.SecurityScheme) {
-	// Parse the claims without validating (since we don't want to bother downloading a key)
-	parser := jwt.Parser{}
-	// we can't know the flow type (implicit, password, application or accessCode) so we choose accessCode for now
-	scheme := spec.OAuth2AccessToken(authorizationURL, tknURL)
-
+func getBearerAuthClaims(bearerToken string) (jwt.MapClaims, bool) {
```

Comment:
what is the bool return value here? like "ok"? confusing with the "assuming authentication required anyway" prints

Reply:
the bool means: found the auth claims.
I will remove the "assuming authentication required anyway"